### PR TITLE
Rewrite the example section.

### LINKED ISSUE #15: Write a "hello world" like example.
The current example is not straightforward enough to express the main concept of this spec., we need a clear one.

### LINKED ISSUE #23: Example 12 uses an undefined setDataFrom() method
Example 12 uses `setDataFrom()` method which is not defined in the spec. I believe this should be `mapDataInto()` instead, correct @kakukogou?

```
// write back to event outputImageBitmap
  outputBitmap.setDataFrom("RGBA32", rgbaBuffer, 0, rgbaBufferLength,
                           bitmap.width, bitmap.height, bitmapPixelLayout.channels[0].stride);
}
```

### PR DESCRIPTION
**Pull request by kakukogou**
I removed all the old examples which used lots of deprecated designs and then I wrote a new simple example which show almost all the usage of all proposed APIs. 

@anssiko and @ChiahungTai, please review it.

Fix #15 and fix #23.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1595,427 +1595,135 @@
...
+          This example demostrates how to hook a video worker to a
+          <a>MediaStreamTrack</a> as a <a>processing video worker</a> and the
+          script code of the video worker shows how to use the extended methods
+          of <a>ImageBitmap extensions</a>
+        </p>
+        <p>
+          The scenario is that captures real-time <a>MediaStream</a> from a
+          web camera, converts the captured color video frames into gray
+          scales and then shows both color and gray videos on the screen. Since
+          the captured color video is likes to be stored as YUV format natively,
```
Comment (anssiko) [inline index.html]: s/likes/likely/

File: index.html [hunk 2]
```diff
@@ -1595,427 +1595,135 @@
...
-        }
-        DisplayTask.prototype = Object.create(Task.prototype);
-        DisplayTask.prototype.constructor = DisplayTask;
-        DisplayTask.TASK_TYPE = "DisplayTask";
-        var numOfWorkers = 2; // initialize the WorkerPool whit this number of workers.
-        var workerPool;
-        var trackid;
-        var frameNum = 0;
-        var dropNum = 0;
+&lt;script type="text/javascript">
```
Comment (anssiko) [inline index.html]: &lt;script&gt; is simpler and valid (no need for type).

File: index.html [hunk 3]
```diff
@@ -1595,427 +1595,135 @@
...
-        ProcessTask.TASK_TYPE = "ProcessTask";
+      <section>
+        <h3>
+          Simple example
+        </h3>
+        <p>
+          This example demostrates how to hook a video worker to a
+          <a>MediaStreamTrack</a> as a <a>processing video worker</a> and the
+          script code of the video worker shows how to use the extended methods
+          of <a>ImageBitmap extensions</a>
```
Comment (anssiko) [inline index.html]: &lt;a href="#imagebitmap-extensions"&gt;

File: index.html [hunk 4]
```diff
@@ -1595,427 +1595,135 @@
...
+        </p>
+        <p>
+          The scenario is that captures real-time <a>MediaStream</a> from a
+          web camera, converts the captured color video frames into gray
+          scales and then shows both color and gray videos on the screen. Since
+          the captured color video is likes to be stored as YUV format natively,
+          this example asks the browser to map the video data into JavaScript
+          <a>ArrayBuffer</a> in any kind of YUV formats which might save the
```
Comment (anssiko) [inline index.html:1616]: Add http://heycam.github.io/webidl/#idl-ArrayBuffer to dependencies after BufferSource.

**Comment by anssiko** [comment]
LGTM! This is a great improvement. It was really good idea to simplify the examples section. This is something people new to the spec will likely look at first.

**Comment by kakukogou** [comment]
Thank you, @anssiko. Here are the modified commits, please take another look!

File: index.html [hunk 5]
```diff
@@ -215,6 +215,11 @@
       </p>
       <p>
         The <code><a href=
+        "http://www.w3.org/TR/WebIDL-1/#idl-ArrayBuffer"><dfn>ArrayBuffer</dfn></a></code>
```
Comment (anssiko) [inline index.html:215]: You can combine these if you like:

The BufferSource and ArrayBuffer are defined in [...]

**Comment by anssiko** [comment]
Please feel free to merge. I had only a minor editorial comment.

**Comment by kakukogou** [comment]
Thank you, @anssiko. Your comments about those editorial conventions are very valuable for me.

Merge this PR and close #15 and #23.

**Pull request merged**